frontend/src/components/user/UserAvatar.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <v-avatar :size="size" :color="color">
+    <span class="white--text" :style="style">{{ initials }}</span>
+  </v-avatar>
+</template>
+<script>
+
+export default {
+  name: 'UserAvatar',
+  props: {
+    size: { type: Number, required: false, default: 48 },
+    user: { type: Object, default: null },
+    campCollaboration: { type: Object, default: null }
+  },
+  computed: {
+    isLoading () {
+      return (this.user || this.campCollaboration)._meta.loading
+    },
+    objectId () {
+      if (this.isLoading) return null
+      if (this.user) { return this.user.id }
+      if (this.campCollaboration) {
+        if (typeof this.campCollaboration.user === 'function') {
+          return this.campCollaboration.user().id
+        }
+        return this.campCollaboration.id
+      }
+      return undefined
+    },
+    objectText () {
+      if (this.isLoading) {
+        return ''
+      }
+      if (this.user) {
+        return this.user.displayName
+      }
+      if (this.campCollaboration) {
```

Comment:
I would put this logic into a computed method which returns the user or null (Option of user).
I don't now about an Option construct, but js has the syntax ?. to avoid NPE and don't write a lot of null checks.

objextText = this.user?.displayName